pkg/ebpf/c/common/context.h

```diff
@@ -209,6 +224,9 @@ statfunc int init_program_data(program_data_t *p, void *ctx)
     }
 
 out:
+    if (unlikely(init_policies_config(p) != 0))
```

Comment:
Not many lines in this function and it is called once, so maybe we should put it inline to simplify?